resteasy-client-api/src/main/java/org/jboss/resteasy/client/jaxrs/i18n/LogMessages.java

```diff
@@ -23,4 +23,8 @@
    @LogMessage(level = Level.DEBUG)
    @Message(id = Messages.BASE + 171, value = "Ignoring exception thrown within InvocationCallback")
    void exceptionIgnored(@Cause Throwable ex);
+
+   @LogMessage(level = Level.ERROR)
+   @Message(id = Messages.BASE + 172, value = "Client side processing failure.")
+   void clientSideProcessingFailure(@Cause Throwable cause);
```

Comment:
I still think that, if the exception is getting re-thrown, as in ClientResponse and ApacheHttpClient43Engine, this should be logged at DEBUG level.